mri/reconstructors/single_channel.py

```diff
@@ -17,16 +17,19 @@
 
 class SingleChannelReconstructor(ReconstructorBase):
     """ This class implements the Single channel MR image Reconstruction.
-    For the Analysis case, finds the solution  for x of:
-        (1/2) * sum(||F x - y||^2_2, 1) + mu * H (W x)
 
-    For the Synthesis case, finds the solution of:
-        (1/2) * sum(||F Wt alpha - y||^2_2, 1) + mu * H (alpha)
+    Notes
+    -----
+        For the Analysis case, finds the solution  for x of:
+        ..math:: (1/2) * sum(||F x - y||^2_2, 1) + mu * H (W x)
+
+        For the Synthesis case, finds the solution of:
+        ..math:: (1/2) * sum(||F Wt alpha - y||^2_2, 1) + mu * H (alpha)
```

Comment:
are all these changes needed for the documentation to be built?

Reply:
.math changes, no, but otherwise these changes weren't showing up in documentation page. Finally, all the white line changes are needed for allowing documentation to build successfully, it's basically white line before parameters etc.

Reply:
fair